cspell.json

```diff
@@ -1843,11 +1857,13 @@
     "SUBSTED",
     "subsytem",
     "subtrees",
+    "successval",
     "sudoer",
     "sudoers",
     "suid",
     "SUPPRESSMSGBOXES",
     "Sutay",
+    "sval",
```

Comment:
Variable names in new code should not be added as spelling exceptions. "mutexes" and "subcategory" a fine because those are English-ish words that are not in the spellchecker's dictionary by default, but anything we can change should just be changed (ie, people's names and things external to this code base that need to be referred to belong in the exceptions list).

Reply:
I'll take those back out, and try to find better names.  for my variables that won't throw errors.  Or is there a way to avoid the spellcheck failure to get cspell to ignore bits of the code?

Reply:
Cspell does the thing a lot of linters do where you can disable with a comment: https://www.npmjs.com/package/cspell#disable-checking, but renaming the variables is best.